src/MasterPeace/Bundle/BookBundle/Entity/Question.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace MasterPeace\Bundle\BookBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Question
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="question")
+ */
+class Question
+{
+    /**
+     * @var integer
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $title;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(type="integer")
+     */
+    private $teacherId;
+
+    /**
+     * @var Book
+     *
+     * @ORM\ManyToOne(targetEntity="Book")
+     */
+    private $book;
+
+    /**
+     * @var Answer
+     *
+     * @ORM\OneToMany(targetEntity="Answer", mappedBy="question")
+     */
+    private $answer;
+
+    /**
+     * @return integer
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTitle()
+    {
+        return $this->title;
+    }
+
+    /**
+     * @param string $title
+     *
+     * @return $this
```

Comment:
ai maniau, kad tik get funkcijose reik nurodyt koks parametras, kazkodel galvojau kad set reikia rasyt @return $this